### DISCUSSION TITLE
是否可以像latex那样在页面下写注释呢

### DISCUSSION THREAD

**Discussion by Lzy-x-x** [Keldos-Li/typora-latex-theme#147] [Q&A] [answered]
是否可以像latex那样在页面下写注释呢

**Comment by Keldos-Li** [comment, marked as answer]
不太能，因为Typora是网页浏览器的逻辑，一串文字就是流式排版，一直流到一个网页的最底下，除非按下「打印」不然根本不知道每一页断在哪里……但是按下「打印」之后又不再支持调整元素的位置了，所以没法把脚注移到每一个打印页面的下方。
